localstack/services/apigateway/apigateway_listener.py

```diff
@@ -467,6 +467,10 @@ def invoke_rest_api_integration(api_id, stage, integration, method, path, invoca
     elif integration_type in ['HTTP_PROXY', 'HTTP']:
         function = getattr(requests, method.lower())
 
+        if uri.__contains__('{proxy}'):
```

Comment:
Thanks for this PR @usmangani1 ! 👍 

Can we add or extend an existing integration test to cover this functionality? Thanks!

(nitpick: Let's better not use `uri.__contains__` directly - better use `if '{proxy}' in uri` )